invenio_communities/assets/semantic-ui/js/invenio_communities/members/members/manager_view/ManagerMembersResultItem.js

```diff
@@ -65,16 +45,22 @@ const dropdownVisibilityOptionsGenerator = (value) => {
 };
 
 export class ManagerMemberViewResultItem extends Component {
+  constructor(props) {
+    super(props);
+    const {result} = props;
+    this.membersApi = new CommunityMembersApi(result.links);
```

Comment:
can we avoid to create this in each ResultItem? I commented the same in @NRodriguezcuellar PR: we should create it once, when the app is created, maybe define it as a singleton, and the use it around.

Reply:
https://github.com/inveniosoftware/invenio-communities/pull/517/files#diff-6bba53592654f0209114cb3c91b07b0485dd5aee906c5ff93bee7c6b0b7ecfabR28 my solution for now.

should it not just pass the `community_pid` ? I see you changed the invitationsApi to use item links but not the membersApi.

Reply:
I was thinking more to create API context and pass it in the context instead of in the props in order to avoid passing it between components